isisd/isis_nb_config.c

```diff
@@ -1881,8 +1884,9 @@ int lib_interface_isis_create(struct nb_cb_create_args *args)
 				__func__, area_tag);
 			abort();
 		}
-
-		circuit = isis_circuit_create(area, ifp);
+		ifp = nb_running_get_entry(args->dnode, NULL, true);
+		if (ifp)
```

Comment:
Unnecessary null check again (`nb_running_get_entry()` will abort the program if it fails).

Reply:
This is resolved.